examples/editor/src/App.tsx

```diff
@@ -20,7 +31,39 @@ function App() {
   // Give tests a way to get prosemirror instance
   (window as WindowWithProseMirror).ProseMirror = editor?._tiptapEditor;
 
-  return <BlockNoteView editor={editor} />;
+  return (
+    <BlockNoteView editor={editor}>
```

Comment:
- I'm not usually a fan of contexts since i feel like they're not as explicit as props, though here I think it's worth it since it would give a cleaner experience for users.
- I would rather have them as children but I don't know if it's possible since you can have multiple children, whereas we should only allow a single child per wrapper. I'll look into it though since I agree it's nicer.
- Also agree with the naming not being that great. I'm not sure what else to call it though since the menu or toolbar element goes inside it. So just calling it e.g. Formatting toolbar would be weird if you also pass a formatting toolbar to it.